Ginger/Ginger/Actions/ActCreatePDFChartEditPage.xaml.cs

```diff
@@ -32,20 +33,27 @@ namespace Ginger.Actions
     public partial class ActCreatePDFChartEditPage 
     {       
         public ActionEditPage actp;
-        private ActCreatePDFChart mAct;
+        private ActCreatePDFChart mAct;        
 
         public ActCreatePDFChartEditPage(ActCreatePDFChart act)
         {     
                 InitializeComponent();
                 mAct = act;
                 Bind();
-                mAct.SolutionFolder = App.UserProfile.Solution.Folder.ToUpper();
+                mAct.SolutionFolder = App.UserProfile.Solution.Folder.ToUpper();                   
         }
-
+                
         public void Bind()
         {
             DataFileNameTextBox.Init(mAct.GetOrCreateInputParam(ActCreatePDFChart.Fields.DataFileName), ActInputValue.Fields.Value);
-            ParamsComboBox.Init(mAct.GetOrCreateInputParam(ActCreatePDFChart.Fields.ParamName), SetParamsCombo(),true);
+            GingerCore.General.ObjFieldBinding(ParamsComboBox.ComboBox, ComboBox.SelectedValueProperty, mAct, ActCreatePDFChart.Fields.ParamName);
+            GingerCore.General.ObjFieldBinding(ParamsComboBox.ComboBox, ComboBox.ItemsSourceProperty, mAct, ActCreatePDFChart.Fields.ParamList);
+            DataFileNameTextBox.ValueTextBox.TextChanged += ValueTextBox_TextChanged;           
+        }
+
+        private void ValueTextBox_TextChanged(object sender, TextChangedEventArgs e)
+        {
+            SetParamsCombo();            
```

Comment:
Add mAct.ParamList = SetParamsCombo here.

Also i Please remove the paramcombo init in Browse_click event